docs/justmat/foulplay/foulplay.md

```diff
@@ -65,6 +61,15 @@ copy to return to normal use.
 
 # extended explanations
 
+### saving/ loading your work
+
+foulplay will automatically save its _current state*_ whenever (a) you switch scripts, or (b) when norns _sleeps_. 
```

Comment:
this is a footnote? aside: i wonder if there's a better way...  i can't recall how these markdown files are getting processed but it'd be nice to figure out something a bit more elegant (and capture it in wiki/FAQ).

Reply:
I'm sure there is a better way. I wanted to include the saving feature in the docs, but as there is no user interaction in saving... I was unsure how to go about it. Should I just omit this part?